lib/HTML/GoogleMaps/V3.pm

```diff
@@ -71,23 +73,27 @@ Valid options are:
 use strict;
 use warnings;
 
-use Geo::Coder::Google;
 use Template;
 
 our $VERSION = '0.09';
 
 sub new {
     my ( $class,%opts ) = @_;
 
-    return bless( {
-        %opts,
-        points     => [],
-        poly_lines => [],
-        geocoder   => Geo::Coder::Google->new(
+    unless($opts{geocoder}) {
```

Comment:
`if ( ! $opts{geocoder} ) { ...` as unless is a horrible construct other than in single postfix conditions :)